### PR TITLE
note on the y coordinate in numpy.meshgrid

### PR DESCRIPTION
**Pull request by RafalSkolasinski**
Adding small explanation why images in continuous coordinates tutorial are upside down.

### CODE REVIEW

**Comment by jbednar** [comment]
Thanks!  Yes, that's definitely a confusing topic.  I'm not completely sure it's clearer after these changes, though.  I'll have to think about whether there's a simpler way to explain all this...

File: doc/Tutorials/Continuous_Coordinates.ipynb [hunk 1]
```diff
@@ -76,7 +76,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now let's build a Numpy array regularly sampling this function at a density of 5 samples per mm: "
+    "``Coords`` function return a ``numpy`` meshgrid that will be helpful for regular sampling this function at a density of 5 samples per mm: "
```
Comment (philippjfr) [inline doc/Tutorials/Continuous_Coordinates.ipynb]: "The ``coords`` function returns``

**Comment by philippjfr** [comment]
Sorry for letting this sit for so long. Any other suggestions on this @jbednar? I'm going through old PRs trying to get them merged or closed. At minimum the grammar I just commented on should be fixed and the PR should be rebased so I can rebuild the test data.

**Comment by jbednar** [comment]
I can't edit the PR, so I'll give @RafalSkolasinski one day to respond, but otherwise we should probably merge it and I can make the changes separately.

File: doc/Tutorials/Continuous_Coordinates.ipynb [hunk 2]
```diff
@@ -87,7 +87,61 @@
...
+   "outputs": [],
+   "source": [
+    "x5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "once values of ``y`` coordinates are incrementing from top to bottom."
```
Comment (jbednar) [inline doc/Tutorials/Continuous_Coordinates.ipynb]: "while values"?

**Comment by philippjfr** [comment]
>I can't edit the PR, so I'll give @RafalSkolasinski one day to respond, but otherwise we should probably merge it and I can make the changes separately.

Probably can't do that because I have to rebuild test data, but I can't do that without rebasing the PR first.

**Comment by jbednar** [comment]
Not sure what you mean; we can merge it without rebasing, can't we?  True, the tests won't pass, bu then we could quickly fix that?

**Comment by philippjfr** [comment]
>True, the tests won't pass, but then we could quickly fix that?

Right, generally try to avoid that, but I guess it's fine.

**Comment by philippjfr** [comment]
Should either merge this or close it since we'll be reorganizing the Tutorials soon.

**Comment by jlstevens** [comment]
It might be worth mentioning #1534 where we are discussing the possibility of making Image and Raster more consistent in future.

**Comment by philippjfr** [comment]
Going to close this PR, the material has moved at this point and we'll have to update the material once we make the planned changes to Raster and Image anyway.

**Pull request closed without merging**